lib/go-tc/totest/tenants.go

```diff
@@ -41,13 +42,13 @@ func DeleteTestTenants(t *testing.T, cl *toclient.Session) {
 	assert.NoError(t, err, "Cannot get Tenants: %v - alerts: %+v", err, tenants.Alerts)
 
 	for _, tenant := range tenants.Response {
-		if tenant.Name == "root" {
+		if tenant.Name == util.Ptr("root") {
```

Comment:
This `if` statement will never execute, `util.Ptr` allocates a new object which will always be different than `tenant.Name`, which is a pointer

Reply:
fixed.